src/index.html

```diff
@@ -2,12 +2,57 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__container">
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="imac retina 2019 silver"
+        >
+
+        <h1 class="card__title">
+          APPLE A1419 iMac 27"
+          Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <p
+          class="card__subtitle">
+          Product code: 195434
+        </p>
+
+        <div class="stars">
+          <div class="stars__container">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="stars__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="price">
+          <div class="price__container">
+            <p class="price__description">Price:</p>
+            <p class="price__number">$2,199</p>
+          </div>
+        </div>
```

Comment:
```suggestion
        <div class="price">
            <p class="price__description">Price:</p>
            <p class="price__number">$2,199</p>
        </div>
```
why do you need two wrappers?